.github/workflows/motion-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Motion Extended Tests
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  check-files:
+    name: Check files
+    runs-on: ubuntu-latest
+    outputs:
+      EXTENDED_MOTION_TEST: ${{ steps.file_checker.outputs.EXTENDED_MOTION_TEST }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event_name == 'pull_request_target' && github.event.pull_request.head.sha || github.ref }}
+          fetch-depth: 0
+      - name: check modified files
+        id: file_checker
+        run: |
+          ( git diff --name-only origin/${{github.base_ref}} | grep -e "motionplan" \
+            -e "referenceframe" \
+            -e "spatialmath" \
+            -e "components\/base\/wheeled" \
+            -e "components\/base\/kinematicbase" \
+            -e "services/motion/builtin" && echo "EXTENDED_MOTION_TEST=true" >> "$GITHUB_OUTPUT" ) || \
+            ( echo "EXTENDED_MOTION_TEST=false" >> "$GITHUB_OUTPUT" )
+
+  motion_tests:
+    name:  Motion Extended Tests
+    needs: check-files
+    runs-on: [buildjet-8vcpu-ubuntu-2204]
+    container: ghcr.io/viamrobotics/rdk-devenv:amd64-cache
+    timeout-minutes: 30
+    steps:
+
+    - name: Check out PR branch code
+      if: github.event_name == 'pull_request_target'
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Check out code in motion-testing repository
+      uses: actions/checkout@v3
+      with:
+        repository: viamrobotics/motion-testing
+        path: motion-testing
+
+    - name: Change ownership to testbot
+      run: chown -R testbot:testbot .
+
+    - name: Run motion tests on PR branch
+      if: ${{ needs.check-files.outputs.EXTENDED_MOTION_TEST == 'true' }}
+      shell: bash
+      env: 
+        URL: ${{ github.event.pull_request.head.repo.html_url }}
+        SHA: ${{ github.event.pull_request.head.sha }}
+      run: |
+        cd motion-testing
+        go mod edit -replace go.viam.com/rdk=${URL#"https://"}@$SHA
+        sudo -Hu testbot bash -lc "go mod tidy && go test ./... -v -run TestMotionExtended"
+
+ 
+    # Now that RDK is public, can't directly comment without token having full read/write access
+    # motion-benchmarks-comment.yml will trigger seperately and post the actual comments
+
+    - name: Prepare code comment
+      run: |
+        echo "PR_NUMBER=${{ github.event.pull_request.number }}" >> pr.env
+
+    - name: Upload results
+      uses: actions/upload-artifact@v3
+      with:
+        name: pr-motion-tests-extended
+        path: |
+          pr.env
+        retention-days: 1
```

Comment:
I'm unclear how this is being used. Nothing appears to download or use this artifact, and the comment workflow mentioned is already used (with a different artifact name) by the existing motion test workflow. Are you removing/replacing the existing test workflow in another PR or something?

Reply:
Ahh, I probably don't need this. I started this PR by copy/pasting a different workflow file